src/main/java/me/dinowernli/grpc/prometheus/Configuration.java

```diff
@@ -100,6 +130,11 @@ public CollectorRegistry getCollectorRegistry() {
     return labelHeaders;
   }
 
+  /** Returns whether or not status code label should be added to latency histogram. */
+  public boolean addCodeLabelToHistograms() {
```

Comment:
The naming sounds like this method modifies state (because it starts with "add"). Can we call this `isAddCodeLabelToHistograms()`?

Granted this would make the grammar a bit sketchy, but it seems worthwhile to clarify that this doesn't modify the configuration.

Reply:
Yes, also thought about that) Do you think is it worth to rename private bool fields also to `isAddCodeLabelToHistograms` (for the sake of consistency) or renaming just public method is better?

Reply:
Meanwhile, replaced globally `addCodeLabelToHistograms` -> `isAddCodeLabelToHistograms`

Reply:
The fields seem less prone to misunderstandings because it's not clear how a field could "do" anything (contrast with a method), so no strong feelings. That said, your consistency argument resonates.